src/main/java/mods/railcraft/common/worldgen/VillagerTrades.java

```diff
@@ -172,4 +171,26 @@ private int stackSize(Random rand, Offer offer) {
             return MathHelper.getRandomIntegerInRange(rand, offer.min, offer.max);
         }
     }
+
+    private static class TrackKitTrade implements EntityVillager.ITradeList {
+
+        private static final List<TrackKit> trackKits;
+
+        static {
+            trackKits = TrackRegistry.TRACK_KIT.getVariants()
+                .values()
+                .stream()
+                .filter(kit -> kit.isEnabled() && kit.isVisible())
+                .collect(Collectors.toList());
+        }
+
+        @Override
+        public void modifyMerchantRecipeList(MerchantRecipeList recipeList, Random random) {
+            ItemStack stack = trackKits.get(random.nextInt(trackKits.size())).getTrackKitItem(2);
```

Comment:
This will crash if trackKits.size() == 0, which might be possible if everything is disabled.